spec/features/quotes/index_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe 'Quote index' do 
+  it 'returns the results for a search' do 
+    visit root_path
+    fill_in :query, with: 'hemingway'
+    click_on 'Search for Quotes'
+    expect(page).to have_content('Total quotes: ')
```

Comment:
For these tests, I would include what gets displayed on the page. So for example here you have
```
'Total quotes: '
```
I would also include the number that gets displayed. So in this particular case it was 2. So it might look like this:
```
'Total quotes: 2'
```
Same with all of your other expects. 
 I would also include the count. Speaking specifically on this `As well as a list of the top 10 quotes that were found`. 
 Your page is indeed listing them but your test could have something like 
```
expect(page).to have_content("Top 10 Quotes:")
expect(page).to have_css('#quote", count: 10)
```